README.md

```diff
@@ -20,7 +20,7 @@ To get started developing, clone the repository and set up your local dev enviro
 
 ```
 git clone https://github.com/bikegriffith/ohiomtb.git
-git checkout parse-react-rewrite
+git checkout react-parse-rewrite
```

Comment:
It’s all good. I was working on my 4th pull request for Hacktoberfest and remembered that detail being slightly off. I was looking at the latest stuff you had in master and it looks very interesting.

Cheers

Kevin Wirth
Pixel Ambassador
216-280-0729
me@wirthdesign.com

> On Oct 31, 2015, at 11:05 PM, Mike Griffith <notifications@github.com> wrote:
> 
> In README.md <https://github.com/bikegriffith/ohiomtb/pull/1#discussion_r43578630>:
> 
> > @@ -20,7 +20,7 @@ To get started developing, clone the repository and set up your local dev enviro
> >  
> >  ```
> >  git clone https://github.com/bikegriffith/ohiomtb.git
> > -git checkout parse-react-rewrite
> > +git checkout react-parse-rewrite
> Actually, I went ahead and merged it all into master, so this whole git checkout step is unnecessary now.
> 
> Were you able to get it running locally?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/bikegriffith/ohiomtb/pull/1/files#r43578630>.
>